### ISSUE TITLE
Setting presets/profiles

### ISSUE THREAD

**Issue by giraffesyo** [invokehub/invokestatic#94]
Suggestion via http://community.invokestatic.com/t/suggestion-save-load-settings/534

Load and save (presumably after #71 is complete) saved settings in a profile-like option.

**Comment by giraffesyo** [comment]
Please review and decide if #wontfix
